aiohttp/base_protocol.py

```diff
@@ -76,8 +79,8 @@ def connection_lost(self, exc: Optional[BaseException]) -> None:
             waiter.set_exception(exc)
 
     async def _drain_helper(self) -> None:
-        if self._connection_lost:
-            raise ConnectionResetError("Connection lost")
+        if not self.connected:
+            raise ConnectionResetError("Not connected")
```

Comment:
```suggestion
            raise ConnectionResetError("Connection lost")
```

Think we should stick to the original, this method doesn't look like it would get called if we didn't create a connection in the first place.